src/pages/patientView/genomicOverview/tracksHelper.ts

```diff
@@ -91,6 +91,10 @@ export const genomeBuilds: Map<string, string> = new Map([
     ['hg38', 'GRCh38'],
     ['38', 'GRCh38'],
     ['mm10', 'GRCm38'],
+    // If the genome build is already Ensembl (e.g. VEP output), make sure the lookup doesn't fail
```

Comment:
There is an old comment above this variable:
```
/**
 * storing chromesome length info
 */
``` 
can be removed (it's about the variable below it, but the comment is redundant imo)